app/views/manage/events/_form.html.haml

```diff
@@ -0,0 +1,9 @@
+= bs_horizontal_simple_form_for @event, url: url_for(action: @event.new_record? ? "create" : "update", controller: "events") do |f|
+  .form-inputs
+    = f.input :title
+    = f.input :description
+    = f.input :location
+    = f.input :start, input_html: {type: "datetime-local"}, html5: true
```

Comment:
Can we incorporate a date-time picker? Wasn't sure what the format of the input should be.

Reply:
see comment about datetime-local below